### PR TITLE
updated air-gapped guide

### PR DESCRIPTION
**Pull request by drodriguez-305**
## Description

Updated and improved a few things in our Air-gapped guide. [Issues-1081](https://github.com/camunda/distribution/issues/149)

- Elasticsearch Helm chart link pointed to deprecated repository updated to proper bitnami repository.
- Camunda Helm chart link updated to helm repository
- Added notes on how the charts should be installed and pointed them to our supported versions.
- Added Artifacthub.io reference
- ([EDIT](https://github.com/camunda/camunda-docs/pull/2931#issuecomment-1833650436)) Docker images hosted in Camunda's DockerHub organization are now mirrored in Camunda's Docker Registry.

## When should this change go live?
No rush just a nice improvement.

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [ ] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by christinaausley** [comment]
Wahoo! Thanks for your contribution @drodriguez-305 -- technical review in latest commit, and one comment for clarification.

Lastly, do you only want this to appear in the `next` version of docs, or also the existing `8.3` docs (and versioned `8.2` and `8.1` docs)? This is found in the `versioned_docs` folder in this repo.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md [hunk 1]
```diff
@@ -33,16 +35,24 @@ The following images must be available in your air-gapped environment:
...
 
-- [Camunda Helm chart](https://github.com/camunda/camunda-platform-helm)
-- [Elasticsearch Helm chart](https://github.com/elastic/helm-charts/tree/main/elasticsearch)
+- [Camunda Helm chart](https://artifacthub.io/packages/helm/camunda/camunda-platform)
+- [Elasticsearch Helm chart](https://github.com/bitnami/charts/tree/main/bitnami/elasticsearch)
 - [Keycloak Helm chart](https://github.com/bitnami/charts/tree/main/bitnami/keycloak)
 - [Postgres Helm chart](https://github.com/bitnami/charts/tree/main/bitnami/postgresql)
 - [Bitnami Common Helm chart](https://github.com/bitnami/charts/tree/main/bitnami/common)
 
+Install the Helm charts by either directly mirroring it, or downloading the artifacts locally. For supported versions, refer to our [supported environments](/reference/supported-environments.md#camunda-8-self-managed) page.
```
Comment (christinaausley) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md:50]: What do you mean by directly mirroring it?

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md [hunk 2]
```diff
@@ -33,16 +35,24 @@ The following images must be available in your air-gapped environment:
...
 
+We currently have a script in the [camunda-helm-respository](https://github.com/camunda/camunda-platform-helm/blob/c6a6e0c327f2acb8746802fbe03b3774b8284de3/scripts/download-chart-docker-images.sh) that will assist in pulling and saving Docker images.
+
 ## Required Helm charts
 
 The following charts must be available in your air-gapped environment:
 
-- [Camunda Helm chart](https://github.com/camunda/camunda-platform-helm)
-- [Elasticsearch Helm chart](https://github.com/elastic/helm-charts/tree/main/elasticsearch)
+- [Camunda Helm chart](https://artifacthub.io/packages/helm/camunda/camunda-platform)
```
Comment (aabouzaid) [inline docs/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md:44]: It's a good idea to use the chart from Artifacthub.io. Let's do that also for Elasticsearch and Keycloak.

Comment (drodriguez-305) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md:44]: Added artifacthub.io links to Elasticsearch and Keycloak.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md (hunk 1 above)
Comment (drodriguez-305) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md]: I have rephrased this thanks!

**Comment by drodriguez-305** [comment]
> Wahoo! Thanks for your contribution @drodriguez-305 -- technical review in latest commit, and one comment for clarification.
> 
> Lastly, do you only want this to appear in the `next` version of docs, or also the existing `8.3` docs (and versioned `8.2` and `8.1` docs)? This is found in the `versioned_docs` folder in this repo.

Confirmed with the team this will be for `next` and `8.3` only. Thanks again!

**Review by akeller — APPROVED** [review]
Giving this a 👍 so we can merge this when ready.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md (hunk 2 above)
Comment (aabouzaid) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md:44]: Also PSQL and Bitnami common.

https://artifacthub.io/packages/helm/bitnami/postgresql
https://artifacthub.io/packages/helm/bitnami/common

**Comment by clementnero** [comment]
As suggested in https://github.com/camunda/camunda-docs/pull/2970#issuecomment-1833627731 I have pushed my changes in this PR.

It is a simple change related to https://github.com/camunda/product-hub/issues/1075 to indicate that the images required for an air-gapped installation, which are hosted on Camunda's DockerHub organization, are also hosted on Camunda's Docker Registry.

File: versioned_docs/version-8.3/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md [hunk 3]
```diff
@@ -33,16 +35,36 @@ The following images must be available in your air-gapped environment:
   - `web-modeler-ee/modeler-webapp`
   - `web-modeler-ee/modeler-websockets`
 
+<<<<<<< HEAD
```
Comment (maxdanilov) [inline versioned_docs/version-8.3/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md:38]: FYI there's cleanup needed from the merge conflict resolution (dropping this line and its counterpart `> > > > > > > 9c4b018e` in the end of the block)

Comment (clementnero) [reply versioned_docs/version-8.3/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md]: Fixed.
I cleaned it but without saving it. Thx @maxdanilov for noticing it 🙇

**Review by felix-mueller — APPROVED** [review]
lgtm besides the 1 comment but perhaps it's clear from context

File: versioned_docs/version-8.1/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md [hunk 4]
```diff
@@ -25,6 +25,15 @@ The following images must be available in your air-gapped environment:
   - `web-modeler-ee/modeler-webapp`
   - `web-modeler-ee/modeler-websockets`
 
+Please note that all the required Docker images, available on DockerHub's Camunda organization, are also provided publicly via Camunda's Docker registry: `registry.camunda.cloud/camunda/<image>`
+
```
[the remark this answers is not in the document]
Comment (clementnero) [reply versioned_docs/version-8.1/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md:29]: Yes. Only Camunda's public images hosted on DockerHub are mirrored on Harbor and are public. This does not include web-modeler, which is private and only available on Harbor.

[the remark this answers is not in the document]
Comment (clementnero) [reply versioned_docs/version-8.1/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md:29]: For me, it's clear from the context, but I can clarify it again if you think it's necessary?

[the remark this answers is not in the document]
Comment (felix-mueller) [reply versioned_docs/version-8.1/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md:29]: no its fine for me too thanks!

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md (hunk 2 above)
Comment (drodriguez-305) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md:53]: also added PSQL and Bitnami common artifacthub links.

File: docs/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md [hunk 5]
```diff
@@ -33,16 +35,24 @@ The following images must be available in your air-gapped environment:
...
-- [Keycloak Helm chart](https://github.com/bitnami/charts/tree/main/bitnami/keycloak)
+- [Camunda Helm chart](https://artifacthub.io/packages/helm/camunda/camunda-platform)
+- [Elasticsearch Helm chart](https://artifacthub.io/packages/helm/bitnami/elasticsearch)
+- [Keycloak Helm chart](https://artifacthub.io/packages/helm/bitnami/keycloak)
 - [Postgres Helm chart](https://github.com/bitnami/charts/tree/main/bitnami/postgresql)
 - [Bitnami Common Helm chart](https://github.com/bitnami/charts/tree/main/bitnami/common)
 
+Install the Helm charts by either making it available on a private registry that can be accessed by the air-gapped environment or downloading the artifacts locally. For supported versions, refer to our [supported environments](/reference/supported-environments.md#camunda-8-self-managed) page.
+
+### Downloading Helm artifacts
```
[the remark this answers is not in the document]
Comment (drodriguez-305) [reply docs/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md]: Removed "Downloading Helm artifacts" section as well.

**Comment by akeller** [comment]
@drodriguez-305 as the assignee for this docs issue, please indicate when you are ready for a final review from DevEx. I didn't realize you were still collecting feedback when I previously reviewed this.

**Comment by drodriguez-305** [comment]
@akeller 

No problem, we are ready now! made the last edits in the last commit. Thank you!

**Comment by christinaausley** [comment]
> > Wahoo! Thanks for your contribution @drodriguez-305 -- technical review in latest commit, and one comment for clarification.
> > Lastly, do you only want this to appear in the `next` version of docs, or also the existing `8.3` docs (and versioned `8.2` and `8.1` docs)? This is found in the `versioned_docs` folder in this repo.
> 
> Hey @christinaausley ,
> 
> Confirmed with the team this will be for `next` and `8.3` only. Thanks again!

Can you confirm here again? Should these changes be removed from `8.2` and `8.1`? If not, I will merge and release.

**Comment by drodriguez-305** [comment]
@christinaausley 
Confirmed only for ```next``` and ```8.3``` nothing else needed to be done.

**Review by christinaausley — APPROVED** [review]

**Pull request merged**